app/uk/gov/hmrc/gform/summary/FormComponentSummaryRenderer.scala

```diff
@@ -0,0 +1,924 @@
+      else {
+        val day = renderMonth(formFieldValidationResult.getCurrentValue(safeId(Date.day)))
+        val month = messages(s"date.$monthKey")
+        val year = formFieldValidationResult.getCurrentValue(safeId(Date.year))
+
+        s"$day $month $year"
+      }
+
+    List(
+      summaryListRow(
+        label,
+        value,
+        None,
+        keyClasses,
+        "",
+        "",
+        if (fieldValue.onlyShowOnSummary)
+          Nil
+        else
+          List(
+            (
+              uk.gov.hmrc.gform.gform.routes.FormController
+                .form(
+                  formTemplateId,
+                  maybeAccessCode,
+                  sectionNumber,
+                  sectionTitle4Ga,
+                  SuppressErrors.Yes,
+                  FastForward.Yes),
+              if (fieldValue.editable) messages("summary.change") else messages("summary.view")))
+      ))
+  }
+
+  private def getTimeSummaryListRows[T <: RenderType](
+    fieldValue: FormComponent,
+    formTemplateId: FormTemplateId,
+    maybeAccessCode: Option[AccessCode],
+    sectionNumber: SectionNumber,
+    sectionTitle4Ga: SectionTitle4Ga,
+    formFieldValidationResult: FormFieldValidationResult
+  )(
+    implicit
+    messages: Messages,
+    lise: SmartStringEvaluator,
+    fcrd: FormComponentRenderDetails[T]): List[SummaryListRow] = {
+
+    val hasErrors = formFieldValidationResult.isNotOk
+
+    val errors = checkErrors(fieldValue, formFieldValidationResult)
+
+    val label = fcrd.label(fieldValue)
+
+    val keyClasses = getKeyClasses(hasErrors)
+
+    val value = if (hasErrors) errors.head.toString else formFieldValidationResult.getCurrentValue.getOrElse("")
+
+    List(
+      summaryListRow(
+        label,
+        value,
+        None,
+        keyClasses,
+        "",
+        "",
+        if (fieldValue.onlyShowOnSummary)
+          Nil
+        else
+          List(
+            (
+              uk.gov.hmrc.gform.gform.routes.FormController
+                .form(
+                  formTemplateId,
+                  maybeAccessCode,
+                  sectionNumber,
+                  sectionTitle4Ga,
+                  SuppressErrors.Yes,
+                  FastForward.Yes),
+              if (fieldValue.editable) messages("summary.change") else messages("summary.view")))
+      ))
+  }
+
+  private def getAddressSummaryListRows[T <: RenderType](
+    formComponent: FormComponent,
+    formTemplateId: FormTemplateId,
+    maybeAccessCode: Option[AccessCode],
+    sectionNumber: SectionNumber,
+    sectionTitle4Ga: SectionTitle4Ga,
+    formFieldValidationResult: FormFieldValidationResult,
+  )(
+    implicit
+    messages: Messages,
+    lise: SmartStringEvaluator,
+    fcrd: FormComponentRenderDetails[T]): List[SummaryListRow] = {
+
+    val hasErrors = formFieldValidationResult.isNotOk
+
+    val errors = checkErrors(formComponent, formFieldValidationResult)
+    val label = fcrd.label(formComponent).capitalize
```

Comment:
This `capitalize` is now acting differently than before. But I'm not sure why it is there in the first place.

Reply:
Thats correct, however I dont think it will make a lot of difference and not sure why its even needed